gary_brown_folder/box_office_challenge.py

```diff
@@ -0,0 +1,47 @@
+
+# this variable is assigned a string
+movies_str = "House on Haunted Hill,Cruel Intentions,10 Things I Hate About You,My Favorite Martian,8MM,Fight Club,The Matrix,Any Given Sunday,The Thin Red Line,A Bug's Life, For Love of the Game, Instinct, Mickey Blue Eyes, The Best Man, Bicentennial Man, The 13th Warrior, October Sky, Lake Placid, Random Hearts, Mighty Joe Young, Superstar, Mystery Men, The Talented Mr. Ripley, Dogma, The Out-of-Towners, The Other Sister, Baby Geniuses, The Story of Us, Blast from the Past, The Insider, Saving Private Ryan, Mysteries of Egypt, The Wood, Arlington Road, T-Rex: Back to the Cretaceous, The Iron Giant, Edtv, At First Sight, The Faculty, Summer of Sam, The Bachelor, Stir of Echoes, Anna and the King, Man on the Moon, Galaxy Quest, Enemy of the State, Waking Ned Devine, Doug's 1st Movie,An Ideal Husband,Everest"
+# this variable is assigned a list of strings
+top_50_list = ['Star Wars: Episode I - The Phantom Menace', 'The Sixth Sense', 'Austin Powers: The Spy Who Shagged Me', 'Toy Story 2', 'The Matrix', 'Tarzan', 'Big Daddy', 'The Mummy', 'Runaway Bride', 'The Blair Witch Project', 'Notting Hill', 'The World Is Not Enough', 'Double Jeopardy', 'Wild Wild West', 'Analyze This', "The General's Daughter", 'American Pie', 'Inspector Gadget', 'Shakespeare in Love', 'Sleepy Hollow', 'The Haunting', 'Patch Adams', 'Entrapment', 'Pokémon: The First Movie - Mewtwo Strikes Back', 'Payback', 'Deep Blue Sea', 'American Beauty', 'The Thomas Crown Affair', 'Stuart Little', 'Blue Streak', 'The Green Mile', 'Bowfinger', 'Life', 'The Bone Collector', "She's All That", 'End of Days', 'Three Kings', 'A Civil Action', 'Stepmom', 'Eyes Wide Shut', 'Never Been Kissed', 'Forces of Nature', 'Varsity Blues', 'Message in a Bottle', "You've Got Mail", 'South Park: Bigger, Longer & Uncut', 'Stigmata', 'Life Is Beautiful', 'The Prince of Egypt',
+               'Deuce Bigalow: Male Gigolo']
+
+
+# TODO: Take the string "movies_str" and convert it into a list and assign it to a variable called "remaining_50_list"
+remaining_50_list = movies_str.split(",")
+
+# Note: Print your new variable to see the list of movies
+print(remaining_50_list)
+
+# TODO: Hmm. Looks like "remaining_50_list" contains the movie "The Matrix" which is also in the "top_50_list". Let's remove this duplicate title from "remaining_50_list".
+remove_item = top_50_list.remove('The Matrix') 
+# Note: Make sure to double check the list you are removing from!
+print(remove_item)
+
+
+# TODO: We're down to 49 movies in "remaining_50_list". Let's add a movie to the list called "Anywhere But Here"
+add_movie = top_50_list.append("Anywhere But Here") 
```

Comment:
You don't need to assign `list.remove()` or `list.append()` to a new variable. Although it works. It's extra code.